tests/pkunit_test.py

```diff
@@ -78,6 +78,16 @@ def test_file_eq_ndiff():
         actual_path=d.join("actual_conformance.ndiff"),
         ndiff_epsilon=1e-9,
     )
+    pkunit.file_eq(
```

Comment:
test the deviance case of passing both rel and abs.  Test the case of passing ndiff_epsilon causing an assertion. 

This particular case names the diff files "rel" but the parameter is "abs". Don't use 1e-13, use something that's not the default.